USF-2278: initial draft of changes for removing references to gdrive and sharepoint (#277)
commerce-docs/microsite-commerce-storefront: src/content/docs/merchants/get-started/terms-and-conditions.mdx

<Tabs>
  <TabItem label="Using EDS placeholders" default>

>     For each agreement, the merchant has to configure into the **`placeholders`** sheet the following information:

Reviewer: ```suggestion
    For each agreement, the merchant has to update the **`placeholders`** sheet with the following information:
```
